UI/Selenium/Utilities/DriverFactory.cs

```diff
@@ -237,6 +237,7 @@ public IWebDriver InitializeSauceDriver(SauceLabsOptions sauceLabsOptions, Sauce
             return WebDriver;
         }
 
+        [Obsolete("Obsolete")]
```

Comment:
Since this project isn't referenced by another application it is probably safe to remove these obsolete methods

Reply:
taken out